src/IssueCreator/IssueManager.cs

```diff
@@ -286,8 +290,10 @@ internal async Task<(bool, string)> TryCreateNewIssueAsync(IssueToCreate issueTo
 
         public async Task<IssueObject> GetIssueAsync(long repoId, int issueNumber)
         {
-            IssueObject issue = await GetValueFromCache(StringTemplate.Issue(repoId, issueNumber), async () => new IssueObject(await _githubClient.Issue.Get(repoId, issueNumber)));
-           return issue;
+            IssueObject issue = await GetValueFromCache(StringTemplate.Issue(repoId, issueNumber), async () => new IssueObject(await _githubClient.Issue.Get(repoId, issueNumber).ConfigureAwait(false)));
+
+            OnIssueLoaded(issue);
```

Comment:
This method is called in three places:
- In the Browse button on the epic assignment screen
- When we find the issue as we leave the text box focus 
- When we load the epics across all the repositories

Should we make this a specific event that is only triggered in a certain condition? Otherwise I worry that we will end up with the value of the s_model changing randomly.

Thoughts?

Reply:
I had the same thought, and I agree it would be worth doing - especially as more functionality is added.

Reply:
Added - bonus, check out the status bar now when the app first starts :)